make.js

```diff
@@ -128,7 +129,7 @@ target.generic = function() {
   mkdir('-p', GENERIC_DIR + '/web');
   mkdir('-p', GENERIC_DIR + '/web/cmaps');
 
-  var defines = builder.merge(DEFINES, {GENERIC: true});
+  var defines = builder.merge(DEFINES, {GENERIC: true, WITH_EDITING: true});
```

Comment:
Setting `WITH_EDITING` to true here is too late.
`target.bundle({});` called previously already checking `WITH_EDITING` and see it is false and excluding `editing/` files